examples/.config/model_params_pytorch.json

```diff
@@ -310,28 +301,19 @@
       "model_src_dir": "recommendation/dlrm/quantization/ptq/eager",
       "dataset_location": "/mnt/local_disk3/dataset/dlrm/dlrm/input",
       "input_model": "/mnt/local_disk3/dataset/dlrm/dlrm/dlrm_weight/tb00_40M.pt",
-      "yaml": "conf.yaml",
-      "strategy": "basic",
-      "batch_size": 16384,
-      "new_benchmark": false
+      "batch_size": 16384
     },
```

Comment:
require new parameter: “main_script”: “xxx.py”,

Reply:
Because test code need to replace some config class, so need to know the name scripts.